core/pkg/model/custom_cost_response.go

```diff
@@ -0,0 +1,241 @@
+package model
+
+import "github.com/opencost/opencost/core/pkg/opencost"
+
+type CustomCostResponse struct {
+	Metadata   map[string]string
+	Costsource string
+	Domain     string
+	Version    string
+	Currency   string
+	Window     opencost.Window
+	Costs      []*CustomCost
+	Errors     []error
+}
+
+type CustomCost struct {
```

Comment:
Would you add godoc strings at least to the structs explaining what they are? E.g. the FOCUS inspiration etc.

Reply:
will do. These will come in via the next PR